common/git-hooks/post-commit

```diff
@@ -0,0 +1,32 @@
+#!/bin/sh
+#
+# Called by "git commit" with no arguments. The hook cannot alter
```

Comment:
I don't think this comment is true anymore, right? It's using the amend capabilities to fix the copyright linter.

Reply:
To a large degree it is still true since the prior commit is already completed before this script is called. I re-worded it to highlight that it cannot alter the success of the prior commit.